docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  db_postgres:
+    image: postgres:16.3
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASS}
+    ports:
+      - "5432:5432"
```

Comment:
Currently, the PostgreSQL container provides the port to the host. If the application connects inside the Docker Compose network, then port-forwarding is not needed.